### ISSUE TITLE
How do I even start using this?

### ISSUE THREAD

**Issue by YJH16120** [inthehand/32feet#109]
I've been browsing documentation for a long time, and don't even know where to start. I want to begin with getting a list of devices that are already connected.

**Comment by peterfoot** [comment]
There isn't a getting started walkthrough/project and I think this is a good request to add to the list. For Bluetooth Classic (InTheHand.Net.Bluetooth) there is more code and documentation in the wild. While some of it was written for the older 3.x version most will still apply. For Bluetooth LE (InTheHand.Bluetooth) there isn't as much yet, although it follows the same model as the Web Bluetooth API and so you may find documentation for that which helps explain how everything fits together.

**Comment by YJH16120** [comment]
I see, but I got pretty tired of just looking at documentation. I think having a quick start section, for Bluetooth Classic or BluetoothLE (or both) will be helpful to most.

**Comment by jcs090218** [comment]
Hi, I have a small demo using this plugin in Unity C#. 

https://github.com/jcs-workspace/Test_32feet

I can work on demo project and write some document for anyone who have trouble using this plugin. :)

**Comment by YJH16120** [comment]
I see. Thank you.

**Comment by CHANG-CHING-CHUNG** [comment]
I understand what you feel @YJH16120 .
I searched a lot of articles online trying to get a small example about how to use 32feet.net to connect both server and client side but I got only code fragments and that didn't help...

I hope the author of this library could make a easy to understand example just like Android did.
[Here is the reference of Android Bluetooth API Overivew](https://developer.android.com/guide/topics/connectivity/bluetooth)

**Comment by YJH16120** [comment]
@CHANG-CHING-CHUNG yeah, I hope he does. Currently I already switched to using the built in namespace `Android.Bluetooth` (which I didn't even know exist). And that works just fine.

**Comment by jcs090218** [comment]
@CHANG-CHING-CHUNG @YJH16120 I have started the an example project with document here, https://github.com/jcs-workspace/32feet.Unity.Example. I am planning on creating more example and document for people to want to use this awesome library!

**Comment by RubenGarcia** [comment]
Please add your examples to the readme.md.
